src/azul/service/elasticsearch_service.py

```diff
@@ -587,6 +587,9 @@ def transform_request(self,
                 es_response = es_search.execute(ignore_cache=True)
             except elasticsearch.NotFoundError as e:
                 raise IndexNotFoundError(e.info["error"]["index"])
+            except elasticsearch.RequestError:
+                final_response = FileSearchResponse({}, {}, {}, entity_type)
```

Comment:
```suggestion
                final_response = FileSearchResponse(hits={},
                                                    pagination={},
                                                    facets={},
                                                    entity_type=entity_type)
```
Just because (almost) everywhere else a `FileSearchResponse` is constructed the keyword arguments are specified.